src/runtime/threads/topology.cpp

```diff
@@ -837,7 +838,7 @@ namespace hpx { namespace threads
     {
         hwloc_bitmap_t cpuset  = mask_to_bitmap(mask, HWLOC_OBJ_PU);
         hwloc_bitmap_t nodeset = hwloc_bitmap_alloc();
-        hwloc_cpuset_to_nodeset_strict(topo, cpuset, nodeset);
+        hwloc_cpuset_to_nodeset(topo, cpuset, nodeset);
```

Comment:
It may not be valid for people using older hwloc. We'll see on the dashboards (though this code probably isn't used much)